src/controllers/borrow.js

```diff
@@ -47,21 +50,45 @@ exports.addUser = (req, response, next) => {
   const userData = req.body;
   setUser(userData)
     .then((results) => {
-      const result = { message: 'User Added !' };
+      const mobileNumberUser = results[0].mobileNumber;
+      const result = { message: 'User Added !', mobileNumberUser };
       response.send(JSON.stringify(result));
     })
     .catch((err) => {
       next(err);
```

Comment:
`next(err)` will render server error page(`HTML`) while your request sent by  `fetch` so you cant render `HTML` files